### ISSUE TITLE
Hyper-parameters settings

### ISSUE THREAD

**Issue by wohaiyo** [cfzd/Ultra-Fast-Lane-Detection#6]
Hi, I use the default config file(culane.py) in your repo to train the model under a single GTX 1080Ti and got an F-measure of 46.1 which is far from your model 69.7. Could you share some hyper-parameters settings?

**Comment by cfzd** [comment]
@wohaiyo 
Hi, the config of `culane.py` is the very one I adopted to get the 69.7 model. According to your description,  the only thing different is I used 4-GPU training. In this case, the batch size is 4x larger. 

Under single GPU training, you can try to decrease the learning rate to 0.1/4 according to Linear Scaling Rule. Or if you have enough GPU memory, you can also try to increase the batch_size to 32 x 4. This kind of setting might be slightly different in terms of BatchNorm, but I think it is negligible for performance.

**Comment by wohaiyo** [comment]
Thanks for your kind advice, it's very helpful !
